web/api/README.md

```diff
@@ -23,12 +18,16 @@ introduced.
 version: `change-api-version.sh 6.40.0`.
 - Update the supported api versions to `6.40` in the server files:
   - `web/codechecker_web/shared/version.py`
-  - `web/server/vue-cli/config/webpack.common.js`
 - Run the command `make build` to generate the Thrift API stubs and to create
-new pypi and npm packages. It will modify the following files:
+new pypi and npm packages. It will modify the following files and
+automatically add it to the git changes:
```

Comment:
```suggestion
automatically add them to the git changes:
```
An alternative phrasing:
```suggestion
automatically stage them to git:
```
According to Git terminology added files are "staged".